packages/@aws-cdk/aws-ecs/test/integ.scheduled-ecs-task.ts

```diff
@@ -0,0 +1,34 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import ecs = require('../lib');
+
+import path = require('path');
+import { ScheduledEc2Task } from '../lib';
+
+const app = new cdk.App();
+
+class ScheduledTaskStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string) {
+    super(scope, id);
+
+    const vpc = new ec2.VpcNetwork(this, 'Vpc', { maxAZs: 1 });
+
+    const cluster = new ecs.Cluster(this, 'EcsCluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', {
+      instanceType: new ec2.InstanceType('t2.micro')
+    });
+
+    new ScheduledEc2Task(this, 'ScheduledTaskExample', {
+      cluster,
+      image: ecs.ContainerImage.fromAsset(this, 'EventImage', {
+        directory: path.resolve(__dirname, 'eventhandler-image')
+      }),
+      scheduleExpression: 'rate(1 minute)',
+      environment: [{ name: 'I_WAS_TRIGGERED', value: 'From CloudWatch Events' }],
+      memoryLimitMiB: 256
```

Comment:
Is this necessary, since the default is already 256? I'd either get ride of it or set a different value.

Reply:
Changing the value